crates/sui-graphql-rpc/src/data/pg.rs

```diff
@@ -124,9 +152,19 @@ mod query_cost {
         };
 
         if cost > max_db_query_cost as f64 {
-            warn!(cost, max_db_query_cost, exceeds = true, "Estimated cost");
+            warn!(
+                cost,
+                max_db_query_cost,
+                exceeds = true,
+                "[Cost] Estimated cost"
```

Comment:
There's no need to make this change.  The reason this code is wrapped in a `query_cost` module is to make sure it gets annotated with "query_cost" in its target.

Reply:
True, but most of the others have a little string preview "[Cost]" so I added it to keep it consistent. I'll review it once it's close to merging.

Reply:
Removed.